### PR TITLE
chore: Prepare next release v2.0.0

### PR DESCRIPTION
**Pull request by vishal423**
- GitHub Actions workflow to auto publish new `npm` package. It requires someone with `admin` access to configure npm token under `NPM_TOKEN` GITHUB secret.
- Update package version to `2.0.0`

### CODE REVIEW

**Comment by vishal423** [comment]
@pascalgrimaud, do we have an org wise `NPM_TOKEN` added under secrets for npm releases? I don't have admin access on this repository, so, can't add mine to do a new release.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Release quarkus blueprint
```
Comment (mraible) [inline .github/workflows/release.yml:1]: I've created a similar workflow in the past. I like to call it "Publish" for the name and filename. When the release is created, it's auto-published to npmjs.org. I'd also recommend switching to two spaces for YAML.

https://github.com/oktadev/schematics/blob/main/.github/workflows/publish.yml

Comment (vishal423) [reply .github/workflows/release.yml:1]: sure, will update

**Comment by vishal423** [comment]
> @vishal423 : no we don't have it because for publishing to NPM, you need an OTP, so the NPM_TOKEN doesn't work

@pascalgrimaud , I don't think 2FA is applicable over here. `svelte` blueprint is configured with similar token (my personal) and that used to work fine. Can you add me to `admins` group so that I can try adding over here as well?

**Comment by pascalgrimaud** [comment]
@vishal423 : just added you in admin quarkus team, so you can give a try

Maybe it's because the project doesn't belong to JHipster NPM organization

**Comment by vishal423** [comment]
@pascalgrimaud , npm auto publish seems to fail due to access denied. Can you please include my name to the allowed list of contributors on npm side?

**Comment by pascalgrimaud** [comment]
@vishal423 : done

**Comment by vishal423** [comment]
Thanks. I can see the publish workflow succeeded this time.

**Comment by vishal423** [comment]
@pascalgrimaud , bounty claimed https://opencollective.com/generator-jhipster/expenses/135272

**Comment by pascalgrimaud** [comment]
@vishal423 : approved

**Pull request merged**